<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FixedLimitsDialog</class>
 <widget class="QDialog" name="FixedLimitsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>250</width>
    <height>150</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>250</width>
    <height>150</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>250</width>
    <height>150</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>231</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Maximum</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Minimum</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="ScientificDoubleSpinBox" name="techSpinBoxMax"/>
    </item>
    <item row="1" column="1">
     <widget class="ScientificDoubleSpinBox" name="techSpinBoxMin"/>
    </item>
   </layout>
  </widget>
  <widget class="QCheckBox" name="checkBoxActive">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>82</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Active</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxZoom">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>80</y>
     <width>82</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Zoom</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxAutoScale">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>80</y>
     <width>82</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Autoscale</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ScientificDoubleSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>ScientificDoubleSpinBox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>FixedLimitsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>FixedLimitsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxAutoScale</sender>
   <signal>clicked(bool)</signal>
   <receiver>techSpinBoxMin</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>177</x>
     <y>91</y>
    </hint>
    <hint type="destinationlabel">
     <x>201</x>
     <y>55</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxAutoScale</sender>
   <signal>clicked(bool)</signal>
   <receiver>techSpinBoxMax</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>178</x>
     <y>86</y>
    </hint>
    <hint type="destinationlabel">
     <x>168</x>
     <y>23</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
